201-openshift-origin-rhel/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "OpenShift Origin on Red Hat Enterprise Linux using Azure On Demand image with Azure Active Directory",
```

Comment:
This should be under 60 chars per guide: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

Please update the PR and see if the build passes. Thanks!